robot_name: "secondary_robot"

# initial coordinates (robot's, not LIDAR's coords!)
start_orange: [120.0, 525.0, 4.71]
start_green: [2880.0, 525.0, 1.57]

# robot size
dim_x: 240.0
dim_y: 240.0
dim_z: 350.0
dim_r: 141.0

# LIDAR scanner coordinates in robot frame
lidar_x: -25.0
lidar_y: 36.0
lidar_a: 1.63

# Manipulators
bee_manipulator_x: -113.0
switch_manipulator_y: 77.0

# Vortex for balls
vortex: [-17.0, -81.0]
